ts/features/bonus/cgn/saga/networking/merchants/cgnMerchantsSaga.ts

```diff
@@ -8,12 +8,91 @@ import {
 } from "../../../../../../utils/errors";
 import { BackendCgnMerchants } from "../../../api/backendCgnMerchants";
 import {
+  cgnMerchantsCount,
   cgnOfflineMerchants,
   cgnOnlineMerchants,
+  cgnSearchMerchants,
   cgnSelectedMerchant
 } from "../../../store/actions/merchants";
 import { withRefreshApiCall } from "../../../../../fastLogin/saga/utils";
 
+export function* cgnGetMerchantsCountSaga(
```

Comment:
since merchant related sagas are in that file I inserted it there.
If we want to refactor this e separate PR would be better for this in my opinion

Reply:
let me know how would you like to proceed

Reply:
I think you can refactor them. If you believe it’s better, you can do it in a new PR

Reply:
I think it would be better to split the functions into a separate file for each one without doing it in another PR since the effort to split these functions isn't enough to create a separate PR.